packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -365,6 +366,14 @@ abstract class FlutterCommand extends Command<void> {
       help: 'Build a JIT release version of your app${defaultToRelease ? ' (default mode)' : ''}.');
   }
 
+  void addFontSubsetFlag() {
+    argParser.addFlag('tree-shake-icons',
```

Comment:
Ah okay. Let's use consistent naming. If the user-facing flag is 'tree-shake-icons', then let's also use those words for the implementation in the code.